daemon/config.go

```diff
@@ -54,6 +55,7 @@ type CommonConfig struct {
 // from the command-line.
 func (config *Config) InstallCommonFlags(cmd *flag.FlagSet, usageFn func(string) string) {
 	cmd.Var(opts.NewListOptsRef(&config.GraphOptions, nil), []string{"-storage-opt"}, usageFn("Set storage driver options"))
+	cmd.Var(opts.NewListOptsRef(&config.AuthZPlugins, nil), []string{"-authz-plugins"}, usageFn("List of authorization plugins by order of evaluation"))
```

Comment:
Evaluation is what is done to something What I think is  really meant is that I can have multiple plugins installed.  A user seeking authorization passes through each one and is evaluated.   Short of writing all that you can change the wording a bit and still convey that. _evaluator n. Agent noun of evaluate; one who evaluates._  

```markdown
List authorization plugins in order from first evaluator to last.
```

or

```markdown
List authorization plugins in order from first authorization check to last.
```